specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/compute.json

```diff
@@ -8938,6 +8938,18 @@
           "$ref": "#/definitions/SubResource",
           "description": "Specifies information about the virtual machine scale set that the virtual machine should be assigned to. Virtual machines specified in the same virtual machine scale set are allocated to different nodes to maximize availability. Currently, a VM can only be added to virtual machine scale set at creation time. An existing VM cannot be added to a virtual machine scale set. <br><br>This property cannot exist along with a non-null properties.availabilitySet reference. <br><br>Minimum api‐version: 2019‐03‐01"
         },
+        "platformFaultDomain": {
```

Comment:
suggesting name change to platformFaultDomainCount to match the mirroring property names

Reply:
@yungezz  is this considered breaking change?  does this need to be in a new Swagger version?

Reply:
platformFaultDomainCount is in VMSS that defines the number of fault domains that can exist.

we need platformFaultDomain on VM that references VMSS to identify which out of possible fault domains, does customer want their VM in.

Reply:
hi @grizzlytheodore according to breaking change guideline https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#api-changes-that-require-a-version-change, adding optional  property to output requires a new version. `platformFaultDomainCount` will be property of `AvailabilitySetListResult`. I'm not sure why breaking change tool doesn't report this. Will sync with tool owner on it.

Reply:
since you're merging to dev branch, it's ok. hi @shpa-microsoft is this PR ready to merge now?